lang/cs/Org.Apache.REEF.Bridge/InteropUtil.cpp

```diff
@@ -129,6 +129,21 @@ JNIEnv* RetrieveEnv(JavaVM* jvm) {
   return env;
 }
 
+
+const int defaultRecursionDepthForExceptionFormat = 3;
+
 String^ FormatJavaExceptionMessage(String^ errorMessage, Exception^ exception) {
-  return String::Concat(errorMessage, Environment::NewLine, exception->StackTrace);
+	return FormatJavaExceptionMessage(errorMessage, exception, defaultRecursionDepthForExceptionFormat);
+}
+
+String^ FormatJavaExceptionMessage(String^ errorMessage, Exception^ exception, int recursionDepth) {
```

Comment:
C++ should allow you to add new parameter `recursionDepth` with default value to existing method instead of declaring a new one.